tidy3d/components/data/monitor_data.py

```diff
@@ -1392,13 +1413,20 @@ class ModeData(MonitorData):
         description="Complex-valued effective propagation constants associated with the mode.",
     )
 
-    n_group_raw: ModeIndexDataArray = pd.Field(
+    n_group_raw: GroupIndexDataArray = pd.Field(
         None,
-        alias="n_group",
+        alias="n_group",  # This is for backwards compatibility only when loading old data
         title="Group Index",
         description="Index associated with group velocity of the mode.",
     )
 
+    dispersion_raw: ModeDispersionDataArray = pd.Field(
+        None,
+        title="Dispersion",
+        description="Dispersion parameter for the mode.",
+        units=PICOSECOND_PER_NANOMETER_PER_KILOMETER,
+    )
+
     @property
     def n_eff(self):
```

Comment:
maybe add a type hint here too while you're at it, thanks

Reply:
Should this be `ModeIndexDataArray` even though it's not complex? Otherwise we need a new data type or an overly-generic `xr.DataArray`.

Reply:
oh wait, I might be a bit confused. maybe just `xr.DataArray` is ok. or `ModeIndexDataArray` is probably ok.. up to you.